ReSharper.FSharp/src/FSharp.Psi/src/CodeFormatter/FSharpCodeFormatterInfoProvider.cs

```diff
@@ -12,8 +15,78 @@ public class FSharpFormatterInfoProvider :
   {
     public FSharpFormatterInfoProvider(ISettingsSchema settingsSchema) : base(settingsSchema)
     {
+      var bindingAndModuleDeclIndentingRulesParameters = new[]
+      {
+        ("NestedModuleDeclaration", ElementType.NESTED_MODULE_DECLARATION, NestedModuleDeclaration.MODULE_MEMBER),
+        ("TopBinding", ElementType.TOP_BINDING, TopBinding.CHAMELEON_EXPR),
+        ("LocalBinding", ElementType.LOCAL_BINDING, LocalBinding.EXPR),
+      };
+
+      var synExprIndentingRulesParameters = new[]
+      {
+        ("ForExpr", ElementType.FOR_EXPR, ForExpr.DO_EXPR),
```

Comment:
Would be great to have some simple tests like checking if the following code is unchanged.

```fsharp
for _ in () do
    ()
```